tutorial/README.md

```diff
@@ -0,0 +1,94 @@
+## Introduction 
+
+This brief, self-contained tutorial will walk the user through an end-to-end run of `mama`. We encourage new users to consult this document prior to using the package on their own data.

```

Comment:
Are the data files you are including here from 1000G or did you mock them up yourself?  We should probably specify a little about that.